src/main/java/core/basesyntax/serviceimpl/ProcessTransactionServiceImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.service.OperationHandler;
+import core.basesyntax.service.ProcessTransactionService;
+import java.util.List;
+
+public class ProcessTransactionServiceImpl implements ProcessTransactionService {
+    private final FruitStrategy fruitStrategy;
+
+    public ProcessTransactionServiceImpl(FruitStrategy fruitStrategy) {
+        this.fruitStrategy = fruitStrategy;
+    }
+
+    @Override
+    public void processData(List<FruitTransaction> data) {
+        for (FruitTransaction entry : data) {
+            OperationHandler service = fruitStrategy.getOperationService(entry.getOperation());
```

Comment:
`OperationHandler service getOperationService` - is it **Handler** or **service** ?


`OperationHandler handler= fruitStrategy.getOperationHandler`
or
`OperationService service = fruitStrategy.getOperationService`